libsql-client/src/cell_value.rs

```diff
@@ -0,0 +1,54 @@
+/// Value of a single database cell
+#[derive(Clone, Debug)]
+pub enum CellValue {
+    Text(String),
+    Float(f64),
+    Number(i64),
+    Bool(bool),
+    Null,
+}
+
+impl std::fmt::Display for CellValue {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            CellValue::Text(s) => write!(f, "\"{s}\""),
+            CellValue::Float(d) => write!(f, "{d}"),
+            CellValue::Number(n) => write!(f, "{n}"),
+            CellValue::Bool(b) => write!(f, "{}", if *b { "1" } else { "0" }),
```

Comment:
Could we use `true` and `false` instead? or is this not supported in our SQL dialect?

Reply:
I thought we can't, but sqlite actually recognizes true and false since 3.23, so let's use it indeed

Reply:
(it's still just syntactic sugar over 0 and 1, but it looks nicer)

Reply:
correction - it works in libSQL, but it won't over wire, because we have a tiny bug at sqld:
```
{"error":"invalid type: boolean `true`, expected a valid SQLite value at line 1 column 49"}
```
I'll report it and meanwhile leave the old code as is